### DISCUSSION TITLE
Pulumi Support in Atmos

### DISCUSSION THREAD

**Discussion by murugslendistry** [cloudposse/community#8] [Feature Requests]
Any plans to support pulumi in Atmos.

**Comment by osterman** [comment]
Pulumi support isn't ruled out, but not currently a focus of ours. It would be interesting to implement support for the new YAML declarative interface of Pulumi as well. If you reach out to me on [LinkedIn](https://linkedin.com/in/osterman), I would like to learn more about how you're using Pulumi to form some ideas on how we can improve it with atmos.
